### DISCUSSION TITLE
Fireship.io Sent Me Here

### DISCUSSION THREAD

**Discussion by emmabyte** [opentffoundation/manifesto#763] [General]
Anyone else hear about this from Fireship?

**Comment by bnjmn21** [comment]
do not click on the star button at github.com/opentffoundation/manifesto

**Reply by cbjsh** [reply]
Great to see over 12k people heeded Jeff's warning.

**Comment by spaceCabbage** [comment]
Duh

**Comment by sharunkumar** [comment]
Guys I think we have to click the star button

**Comment by M-Hassan-Raza** [comment]
I heard I'm being rude to a billion-dollar corporation if I click on the Star button. And that's exactly what I did.

**Comment by s5msepi0l** [comment]
nothing makes me more happy than 10k+ people giving this repo a star

**Comment by UKn0wWho** [comment]
Watching this grow is epic XD, I saw the video as soon as it was released and I started ~6k
